fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_ATC_DepartReq.js

```diff
@@ -34,6 +34,7 @@ class CDUAtcDepartReq {
 
     static ShowPage1(mcdu, store = CDUAtcDepartReq.CreateDataBlock()) {
         mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.CDUAtcDepartReq;
```

Comment:
I think this has not been consistently done, but the pages should be defined here: https://github.com/flybywiresim/aircraft/blob/master/fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js#L80

Otherwise `mcdu.page.CDUAtcDepartReq` would just be undefined or am I missing something?

Reply:
Oopsie 😅  and in typical js fashion it works. I'll change it

Reply:
Yeah, I don't blame you 😬 Cool!

Reply:
Changed :)